packages/backend/src/tokens/keys.ts

```diff
@@ -208,7 +196,19 @@ async function fetchJWKSFromBAPI(apiUrl: string, key: string, apiVersion: string
 }
 
 function cacheHasExpired() {
-  return Date.now() - lastUpdatedAt >= MAX_CACHE_LAST_UPDATED_AT_SECONDS * 1000;
```

Comment:
@nikosdouvlis I agree that we should align all of them to be in MS. It was capped at 5 minutes because we considered that it would be safe to consider that in 5 minutes all the systems related to the JWKs would have been updated to include a new instance or an update to the JWKs initiated by our Clerk API (eg Edge cache, read-only replicas, ...)

Reply:
That makes sense but we probably need to document jwksCacheTtlInMs better if we don't deprecated it as the prop is somewhat confusing